utils/templates/test_module.yml.in

```diff
@@ -10,6 +10,7 @@
   - name: Ensure $name NAME is absent
```

Comment:
Line 3 is not correct. It should be hosts: "{{ ipa_test_host | default('ipaserver') }}"
Do we also need gather_facts: true ?

Reply:
I prefer that we use `gather_facts: no` unless it is required. Test playbook should also serve as documentation, and showing that "this is not required for these cases" is good, in my opinion. 

Fixed line 3.